### DISCUSSION TITLE
Context / Documentation

### DISCUSSION THREAD

**Discussion by kamalkech** [oscartbeaumont/rspc#60] [rspc Help] [answered]
if i follow the documentation i found a looot of bugs, my question is, this plugins in dev mode or what ??? because i spent a loooot of time to test and any thing works good, check this capture plz

![image](https://user-images.githubusercontent.com/1313428/192169214-e5be9957-2039-4e78-8734-b8f4717a9130.png)

**Comment by oscartbeaumont** [comment, marked as answer]
It looks like you're using the new syntax (currently on the main branch and undocumented - `|t| t(...)`) with an older version of the library (published to crates.io). I am still half a week off releasing the new syntax so I would probally reccomend using it but it won't be documented.

To use the newer version set the following in your `Cargo.toml`:
```toml
rspc = { git = "https://github.com/oscartbeaumont/rspc", branch = "main", features = ["axum"] }
```

Then continue using the code you have avoid from the `examples` folder in the repository.

If you want to continue using the crates.io version you will have to refer to the documentation website for the older syntax as all of the examples in the repository have been upgraded.

Sorry about the inconvenience of this change. The next rspc release has got a lot of breaking changes coming to make the project mor extensible in the future.

**Reply by oscartbeaumont** [reply]
I don't know if it helps but I have partially written the migrations guide for the next release [here](https://0-0-6.rspc-docs.pages.dev/breaking-changes). Not all of the changes shown in that guide are on the master branch so I would use it as a reference if you run into an error with something.
